Assets/_Project/Scripts/Weapons/Projectile.cs

```diff
@@ -33,9 +32,10 @@ void DestroySelf()
 
         void OnTriggerEnter(Collider co)
```

Comment:
We should be raycasting every frame instead of relying on colliders. The collisions are very unreliable with this method

Reply:
That being said, this particular request is orthogonal to your change, so we can make a card for it on Trello and fix it in a separate PR